tests/functional/base_test.go

```diff
@@ -82,6 +85,15 @@ func CreateNames(placementAPIName types.NamespacedName) Names {
 		RoleBindingName: types.NamespacedName{
 			Namespace: placementAPIName.Namespace,
 			Name:      "placement-" + placementAPIName.Name + "-rolebinding"},
+		CaBundleSecretName: types.NamespacedName{
+			Namespace: placementAPIName.Namespace,
+			Name:      "combined-ca-bundle"},
```

Comment:
maybe rename the const and use them here as well and not just in the GetTLSPlacementAPISpec?